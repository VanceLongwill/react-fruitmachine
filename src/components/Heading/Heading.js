import React from 'react'
import styled from 'styled-components'

const HeadingContainer = styled.div`
  text-align: center;
`
const Title = styled.h1`
  color: rgb(75, 198, 155)
`

const Subtitle = styled.h2`
  color: rgb(68, 68, 68);
`

export function Heading() {
  return (
    <HeadingContainer>
      <Title>Fruit Machine</Title>
      <Subtitle>Click the lever to play!</Subtitle>
    </HeadingContainer>
  )
}
