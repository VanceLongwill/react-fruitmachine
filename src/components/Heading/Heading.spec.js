import React from 'react'
import { mount } from 'enzyme'

import { Heading } from './Heading'

describe('Heading', () => {
  it('should match snapshot', () => {
    expect(mount(<Heading />)).toMatchSnapshot()
  })
})
