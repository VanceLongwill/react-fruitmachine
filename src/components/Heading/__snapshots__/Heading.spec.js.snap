// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match snapshot 1`] = `
<styled.div>
  <div
    className="sc-bdVaJa khUSez"
  >
    <styled.h1>
      <h1
        className="sc-bwzfXH dLfjDn"
      >
        Fruit Machine
      </h1>
    </styled.h1>
    <styled.h2>
      <h2
        className="sc-htpNat iERxhB"
      >
        Click the lever to play!
      </h2>
    </styled.h2>
  </div>
</styled.div>
`;
